src/browser/api/browser_view.ts

```diff
@@ -89,12 +105,21 @@ export async function attach(ofView: OfView, toIdentity: Identity) {
         }
         windowCloseListenerMap.get(ofWin).set(ofView, listener);
         updateViewTarget(ofView, toIdentity);
+        of_events.emit(route.view('attached', ofView.uuid, ofView.name), {
```

Comment:
Should previousTarget be included on the first attach call (during creation)?

Reply:
I think so. On creation, `previousTarget` === `target`, so a user could potentially use that to take different actions for a creation `attach` and an attachment that happens from one window to another (`previousTarget` !== `target`). There could easily be another way to do this though, and also we could question just how useful this is for users.